sdk/containerregistry/azure-containerregistry/tests/test_container_registry_client.py

```diff
@@ -624,13 +641,11 @@ def test_upload_oci_manifest(self, containerregistry_endpoint):
         
         client.delete_manifest(repo, digest)
 
-    @pytest.mark.live_test_only
     @acr_preparer()
+    @recorded_by_proxy
     def test_upload_oci_manifest_stream(self, containerregistry_endpoint):
-        # TODO: remove the "@pytest.mark.live_test_only" annotation once moved to the new test framework
-        # Arrange
         repo = self.get_resource_name("repo")
-        base_path = os.path.join(self.get_test_directory(), "data", "oci_artifact")
+        base_path = os.path.join(os.getcwd(), "data", "oci_artifact")
```

Comment:
We should use the location of the file. Reference [this]() for an example.

For the failure of this test specifically, are you running `pytest` out of the package directory, or the tests directory? Because I think the reason you don't see any failures locally is that you're invoking pytest in the context of the `test` directory, so wo when it does the path join, it _can_ retrieve the `manifest.json.` You can see that by the expected vs actual paths.

Expected:
`sdk/containerregistry/azure-containerregistry/data/oci_artifact/manifest.json`
Actual:
`sdk/containerregistry/azure-containerregistry/tests/data/oci_artifact/manifest.json`


In CI, it's running pytest from your _package_ directory. `tox -e whl -c ../../../eng/tox/tox.ini` after installing `tox tox-monorepo` should reveal the same failure.